packages/babel-plugin-transform-bigint/test/fixtures/addition/expected.js

```diff
@@ -0,0 +1,2 @@
+import _BigInt from "bigint-polyfill";
```

Comment:
I had been thinking of this for the polyfill so that we could get browser support as well https://github.com/MikeMcl/bignumber.js/ @littledan are there any drawbacks? i'm not really familiar with the different libraries in this space